getting-started/templates/Openshift-GCP/os_gcp_supplemental_values.yaml

```diff
@@ -0,0 +1,286 @@
+        openshift:
+          adaptSecurityContext: disabled
+
+testmonitorservice:
+  ## Configuring sidecars for the Test Monitor Service
+  sidecars:
+    - name: cloud-sql-auth-proxy
+      image: gcr.io/cloud-sql-connectors/cloud-sql-proxy:2.8.0
+      volumeMounts:
+        - name: <token-volume-name> #<ATTENTION> Enter the volume name where the token is mounted
+          readOnly: true
+          mountPath: /var/run/secrets/openshift/serviceaccount/
+        - name: <config-volume-name> #<ATTENTION> Enter the volume name where config.json is mounted
+          mountPath: /secrets/
+          readOnly: true
+      env:
+        - name: "GOOGLE_APPLICATION_CREDENTIALS"
+          value: /secrets/service_account.json
+      args:
+        # If connecting from a VPC-native GKE cluster, you can use the
+        # following flag to have the proxy connect over private IP
+        - "--private-ip"
+        # - "--auto-iam-authn"
+
+        # Enable structured logging with LogEntry format:
+        - "--structured-logs"
+        - "--exit-zero-on-sigterm"
+
+        # Replace DB_PORT with the port the proxy should listen on
+        - "--port=5432"
+        - "<connection-name>" #<ATTENTION> Enter the connection name from the CloudSQL instance
+      # - "--credentials-file=/var/run/secrets/openshift/serviceaccount/token"
+      securityContext:
+        # The default Cloud SQL Auth Proxy image runs as the
+        # "nonroot" user and group (uid: 65532) by default.
+        runAsNonRoot: true
+      # restartPolicy: Always
+      # You should use resource requests/limits as a best practice to prevent
+      # pods from consuming too many resources and affecting the execution of
+      # other pods. You should adjust the following values based on what your
+      # application needs. For details, see
+      # https://kubernetes.io/docs/concepts/configuration/manage-resources-containers/
+      resources:
+        requests:
+          # The proxy's memory use scales linearly with the number of active
+          # connections. Fewer open connections will use less memory. Adjust
+          # this value based on your application's requirements.
+          memory: "1Gi"
+          # The proxy's CPU use scales linearly with the amount of IO between
+          # the database and the application. Adjust this value based on your
+          # application's requirements.
+          cpu: "1"
+
+  ## Volumes that can be used in extra containers
+  extraVolumes:
+    - name: <token-volume-name> #<ATTENTION> Enter the volume name where the token is to be mounted
+      projected:
+        sources:
+          - serviceAccountToken:
+              audience: openshift
+              expirationSeconds: 3600
+              path: token
+    - name: <config-volume-name> #<ATTENTION> Enter the volume name where config.json is to be mounted
+      secret:
+        secretName: <secret-name> #<ATTENTION> Enter the secret name where config.json going to be added.
```

Comment:
We don't want secrets in the same values file as the rest of the configuration, so we can encrypt the secrets or otherwise keep them out of source control. For the purpose of this getting started template that would mean separate values and secret values files that you'd have the customer fill out separately and combine when running the helm deploy.

Reply:
The secrets will not be added to the values file. The secret mentioned here is just a placeholder which will point to the config file.